MyBus/Podfile.lock

```diff
@@ -63,6 +66,6 @@ SPEC CHECKSUMS:
   RealmSwift: 838058b2db95b12cb86bd0cf209df642c33fb60a
   SwiftyJSON: 04ccea08915aa0109039157c7974cf0298da292a
 
-PODFILE CHECKSUM: c24202a2d7b27649900b2a9188d325a3a01b443c
+PODFILE CHECKSUM: 84c6e39b23171959e648ef6dee53a4771c76d847
 
-COCOAPODS: 1.1.1
+COCOAPODS: 1.0.1
```

Comment:
I believe that it won't affect existing Pods. I updated in #145 only because I have installed the latest gem of CocoaPods